phcovid/phcovid.py

```diff
@@ -52,11 +68,30 @@ def parse_numeric(s):
     return case_list
 
 
-def get_cases(
-    rename_dict=RENAME_DICT, val_alias=VAL_ALIAS, none_alias=NONE_ALIAS
-):
+def attach_supplement_data(dataframe):
+    # change targets to extend the dataset
+    targets = [
```

Comment:
can we set `targets` as an argument and just set it as this list by default? I expect there are more fields that we'll want to use. E.g. the long lat fields from the spreadsheet are apparently more complete.

Reply:
Got it! Moved targets to `constants.py` and imported that to make it as a parameter for `attach_supplement_data` function.

Reply:
Decided to remove the targets list constant from `constants.py` and `phcovid.py` and instead define what you need to extract to make it straightforward code-wise.

```python
df_aliased[["status", "symptoms"]] = extract_supplement_data(
    df_aliased,
    targets=["case no.", "status", "symptoms"]
)
```

This will also solve the other problems for inplace DF modifications.